messages/commands/deposit.go

```diff
@@ -13,3 +19,21 @@ type ApproveDeposit struct {
 	AccountID     string
 	Amount        int64
 }
+
+// MessageDescription returns a human-readable description of the message.
+func (m *Deposit) MessageDescription() string {
+	return fmt.Sprintf(
+		"depositing %s into account %s",
```

Comment:
It might get kind of long and wordy unless I put it in brackets or something?
```go
"txn %s depositing %s into account %s",
"depositing %s into account %s for txn %s",
"[%s] depositing %s into account %s",
"depositing %s into account %s [%s]",
```

Reply:
The second one read pretty well to me, even without abbreviating: `depositing %s into account %s for transaction %s`.

It's definitely nice when they work out really concise, but I think since they are basically a replacement for manual logging it's okay if they do get a little longer. That text certainly wouldn't seem "too long" for a log line.

Reply:
It does get hard to keep it all in a single sentence though. Do I repeat the word "for"?
Original:
`consumed %s from daily debit limit for account %s`
With transaction:
`consumed %s from daily debit limit for account %s for transaction %s`.

Maybe I should use "of account", what do you think about this?
`consumed %s from daily debit limit of account %s for transaction %s`.

And there are longer ones when there are "error" cases like these:
`consuming %s from daily debit limit for account %s was denied for exceeding limit`
`declining transfer of %s from account %s to account %s: %s` _(reason)_

Would those "error" cases become something like this?
`consuming %s from daily debit limit of account %s for transaction %s was denied for exceeding limit`
`declining transfer of %s from account %s to account %s for transaction %s: %s` _(reason)_

Reply:
> Maybe I should use "of account", what do you think about this?
> `consumed %s from daily debit limit of account %s for transaction %s`.

I like this.

> Would those "error" cases become something like this?
> `consuming %s from daily debit limit of account %s for transaction %s was denied for exceeding limit`
> `declining transfer of %s from account %s to account %s for transaction %s: %s` _(reason)_

Which message types are these for? In the case of ones that know it's for a "transfer", you probably don't need to put ALL the information from the message. It might be enough to say:

`declining transfer 123123: daily debit limit exceeded`

Reply:
The `consuming %s from daily debit limit`... happens during withdrawal and transfer transaction events.

Reply:
Ok, so how about these then:

- `DailyDebitLimitConsumed` - `transaction 123123 consumed $5.50 from daily debit limit of account 456456`
- `DailyDebitLimitExceeded` - `transaction 123123 exceeded daily debit limit of account 456456 by $0.25`

Reply:
Ideas for these?
- `transaction %s is depositing %s into account %s` (kind of makes sense)
- `transaction %s is approving deposit of %s into account %s` (doesn't really make sense)

Alternate idea for number 2 above:
- `transaction %s is being approved for deposit of %s into account %s` (still doesn't really sound right)

Reply:
Would you mind putting the event type names in front of them?

Reply:
It doesn't feel quite right to me to say that the "transaction is doing it", so I'd suggest:

- `command/Deposit` - `depositing %s into account %s for transaction %s` 
- `command/ApproveDeposit` - `approving deposit of %s into account %s for transaction %s`

Reply:
Nope, I messed them up. One sec.

Reply:
See Slack, this comment chain is nuts :)

Reply:
After discussion on Slack, this has now been updated.